### PR TITLE
Fix graphene framework details

### PR DESCRIPTION
**Pull request by lrafeei**
Co-authored-by: Timothy Pansino <TimPansino@users.noreply.github.com>
Co-authored-by: Uma Annamalai <umaannamalai@users.noreply.github.com>

_Before contributing, please read our [contributing guidelines](https://github.com/newrelic/newrelic-python-agent/blob/main/CONTRIBUTING.rst) and [code of conduct](https://github.com/newrelic/.github/blob/master/CODE_OF_CONDUCT.md)._

# Overview
Describe the changes present in the pull request

# Related Github Issue
Include a link to the related GitHub issue, if applicable

# Testing
The agent includes a suite of tests which should be used to
verify your changes don't break existing functionality. These tests will run with
Github Actions when a pull request is made. More details on running the tests locally can be found
[here](https://github.com/newrelic/newrelic-python-agent/blob/main/CONTRIBUTING.rst#testing-guidelines),
For most contributions it is strongly recommended to add additional tests which
exercise your changes.

### CODE REVIEW

File: newrelic/hooks/framework_graphql.py [hunk 1]
```diff
@@ -450,15 +444,11 @@ def wrap_graphql_impl(wrapped, instance, args, kwargs):
...
-            from graphene.types.schema import Schema as GrapheneSchema
-            if isinstance(schema, GrapheneSchema):
-                trace.product = "Graphene"
-                framework = graphene_framework_details()
-                transaction.add_framework_info(name=framework[0], version=framework[1])
-        except ImportError:
-            pass
+        # Handle Schemas created from frameworks
+        if hasattr(schema, "_nr_framework"):
+            trace.product = "Graphene"
```
Comment (TimPansino) [inline newrelic/hooks/framework_graphql.py:449]: Can we move this line down a line, and change the hard coded graphene to use `framework[0]`?

File: tox.ini [hunk 2]
```diff
@@ -117,8 +117,8 @@ envlist =
     ;temporarily disabling tests on flask master
     ; python-framework_flask-{py37,py38,py39,py310,pypy3}-flask{latest,master},
     python-framework_flask-{py37,py38,py39,py310,pypy3}-flask{latest},
-    python-framework_graphene-{py27,py36,py37,py38,py39,py310,pypy,pypy3}-graphenelatest,
-    python-framework_graphene-py37-graphene{0200,0201},
+    python-framework_graphene-{py36,py37,py38,py39,py310}-graphenelatest,
```
Comment (umaannamalai) [inline tox.ini:120]: Was there an incompatibility issue for pypy, pypy3, and py27 with the latest version of graphene? We probably want to test some version of pypy and Python 2.7 with newer graphene versions to check for breaking changes (if possible).

Comment (lrafeei) [reply tox.ini:120]: It seems that Graphene 3 only supports python 3.6, 3.7, and 3.8 (but also works for 3.9 and 3.10)

File: newrelic/hooks/framework_graphql.py (hunk 1 above)
Comment (lrafeei) [reply newrelic/hooks/framework_graphql.py:449]: Sure

File: tox.ini (hunk 2 above)
Comment (lrafeei) [reply tox.ini:120]: But Graphene 2 still works/is supported with the newer versions of python, so I've added those into the Graphene 2 section of the tox file

**Review by TimPansino — APPROVED** [review]

**Pull request merged**